### DISCUSSION TITLE
disable "daycore" sfx when slowing down in editor?

### DISCUSSION THREAD

**Discussion by NafeeJ** [ppy/osu#23789] [Q&A] [answered]
when you reduce the playback speed in the editor, it applies some kind of a daycore effect which makes it difficult to hear the song details as opposed to stable.

can you disable this?

**Comment by peppy** [comment, marked as answer]
This was a design choice. The alternative to this actually creates perceptual timing issues with the playback of the track.

But we hear your feedback and may change this in the future.

**Reply by furiner** [reply]
i find it much harder to listen to the deeper parts of a song due to the lower pitching in comparsion to stable; especially w/ drums

maybe it could be made an option?

**Reply by peppy** [reply]
Potentially yes.

**Reply by JarvisGaming** [reply]
Taiko mapping is highly based on pitch, so having playback rate screw with the pitch of the song makes complex songs very difficult to map in lazer.
